src/main/java/com/rocketden/main/service/GameService.java

```diff
@@ -40,6 +40,9 @@ public RoomDto startGame(String roomId, StartGameRequest request) {
 			throw new ApiException(RoomError.INVALID_PERMISSIONS);
 		}
 
+		room.setIsActive(true);
+		repository.save(room);
+
 		RoomDto roomDto = RoomMapper.toDto(room);
 
 		String socketPath = String.format(START_GAME_SOCKET_PATH, roomId);
```

Comment:
Suggestion: change this to send the socket update on the usual endpoint (`subscribe-user`), that way we don't need a separate endpoint just for starting a game